### DISCUSSION TITLE
Fragen und Probleme

### DISCUSSION THREAD

**Discussion by 1fexd** [eGovPatchesAT/id-austria#1] [General]
Diskussion für Fragen und Probleme mit dem Patch

**Comment by opg2000** [comment]
was bedeutet "gist"

**Reply by 1fexd** [reply]
![image](https://github.com/eGovPatchesAT/id-austria/assets/58902674/3a564396-261a-4c91-99f5-ee86cfcebc44)

**Comment by opg2000** [comment]
....aha... :)

**Comment by raphatobo** [comment]
Servus, funktioniert das ganze auch unter e/OS?

**Reply by 1fexd** [reply]
Ja

**Reply by raphatobo** [reply]
Danke für die Rückmeldung, das war das letzte was mich davon abgehalten hat e/OS zu installieren!

**Comment by birdeee** [comment]
Ich habe soeben versucht die aktuellste Version von ID Austria (5.0.0) mit ReVanced zu patchen erhalte aber beim root detection patch einen Fehler dass der fingeprint nicht übereinstimmt
Ich nehme mal an dass dafür der Patch an die neueste Version angepasst werden müsste - korrekt? oder kann es sein dass ich etwas falsch mache?

vielen dank jedenfalls für die Info zu dem Thema!

**Reply by 1fexd** [reply]
Ja, die neue Version ist aktuell nicht unterstüzt

**Comment by Herbi-1966** [comment]
Hallo,
bei mir bricht die gepatchte ID-Austria 5.0.0 gleich nach dem Start ab, da Root erkannt.
Bin auch mit zusätzlichen Magisk Versuchen () nicht weiter gekommen ...

**Reply by 1fexd** [reply]
Ja, die neue Version hat bessere Checks

**Comment by mcbabo** [comment]
Hallo,
wenn ich OEM wieder sperre, muss ich trotzdem die App patchen? (https://grapheneos.org/install/web#disabling-oem-unlocking)
Könnte ich auch eine "Sandboxed Play Services" einrichten und über PlayStore die App normal herunterladen?

**Reply by 1fexd** [reply]
Nein, muss auf GrapheneOS nicht gepatcht werden

**Reply by mcbabo** [reply]
Update: App funktioniert auf GrapheneOS, meldet aber einen Falschalarm und sagt Bootloader ist unlocked obwohl Bootloader locked ist.

**Comment by mrpppatrick** [comment]
Ich verwende ein Pixel 4a mit Pixelbuilds custom Rom, 
habe die aktuelle Vers. 5.1.0 mit Revanced gepatched, trotzdem bekomme ich bei der Einrichtung von ID Austria eine Fehlermeldung 
<img width="320" height="636" alt="Screenshot_20251009-150955 1" src="https://github.com/user-attachments/assets/5f9e8648-e26d-4063-9bac-93c178c5d9a2" />

gibt es irgendeine alternative Möglichkeit ID Austria mit custom Rom zu verwenden?

**Reply by 1fexd** [reply]
Hallo, ab Version `5.0.0` lässt sich die App leider nichtmehr einfach patchen.

**Reply by raoulbhatia** [reply]
Patchen ist nicht mehr nötig, vgl. https://github.com/eGovPatchesAT/id-austria/issues/3#issuecomment-3146684208
Es reicht, die App in der Magisk DenyList anzuhaken.

Ich verwende Version 5.1.0 auf einem Google Pixel 6a mit LineageOS.
Ob das auch für ältere Geräte funktioniert, kann ich leider nicht sagen.
